sceptre/bridge/config/prod/bridge-ops-djeng.yaml

```diff
@@ -0,0 +1,16 @@
+template:
+  type: http
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.4.0/EC2/jc-ec2-linux.j2
+stack_name: jc-ec2-linux
+dependencies:
+  - prod/bootstrap.yaml
+parameters:
+  JcUserId: dwayne.jeng@sagebase.org
+  KeyName: bridge-ops
+  InstanceType: t3a.micro
+  JcConnectKey: !ssm /infra/JcConnectKey-djeng
+  JcServiceApiKey:
+  JcSystemsGroupId:
```

Comment:
Not sure what I'm supposed to put for these attributes.

Reply:
These are to allow access via jumpcloud & VPN.  You guys already have full  access to bridge accounts so I'm wondering if these things are needed at all?

The `jc-ec2-linux.j2` template was specifically created for scicomp provisioning and to allow scientist access to the instance without giving them access to the AWS account.  I think this template probably does not fit your use case?

Reply:
The use case is that we need to be able to SSH into this EC2 machine and run scripts from the command-line. We also want to ensure the EC2 instance is accessible only from VPN.

Reply:
the `basic-ec2.j2` templates does not require setting `JcUserId`, `KeyName`, `JcConnectKey`, `JcServiceApiKey`, `JcSystemsGroupId`,  or `AMIId` parameters.

Once provisioned you can access the instance using the [AWS SSM session manager](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2632286259/AWS+SSM+Session+Manager#Access-to-a-private-EC2-instance%3A) instead of the VPN.